docs/connectors/mongodb.asciidoc

```diff
@@ -190,14 +190,25 @@ Every change event for the `customers` collection will feature the same key stru
       ]
     },
     "payload": {
-      "_id": "1004"
+      "id": "1004"
     }
   }
 ----
 
-The `schema` portion of the key contains a Kafka Connect schema describing what is in the payload portion, and in our case that means that the `payload` value is not optional, is a structure defined by a schema named `fulfillment.inventory.customers.Key`, and has one required field named `_id` of type `string`. If we look at the value of the key's `payload` field, we'll see that it is indeed a structure (which in JSON is just an object) with a single `_id` field, whose value is a string containing the integer `1004`.
+The `schema` portion of the key contains a Kafka Connect schema describing what is in the payload portion, and in our case that means that the `payload` value is not optional, is a structure defined by a schema named `fulfillment.inventory.customers.Key`, and has one required field named `id` of type `string`. If we look at the value of the key's `payload` field, we'll see that it is indeed a structure (which in JSON is just an object) with a single `id` field, whose value is a string containing the integer `1004`.
 
-This example used a document with an integer identifier, but any valid MongoDB document identifier (including documents) will work. The value of the `_id` field in the payload will simply be the string representation of that value.
+This example used a document with an integer identifier, but any valid MongoDB document identifier (including documents) will work. The value of the `id` field in the payload will simply be a string representing a MongoDB extended JSON serialization (strict mode) of the original document's `_id` field. Find below a few examples showing how `_id` fields of 
+different types will get encoded as the event key's payload:
+
+* `_id` of type integer e.g. 1234 -> `{ "id" : "1234" }`
+* `_id` of type float e.g. of 12.34 -> `{ "id" : "12.34" }`
+* `_id` of type string e.g. "1234" -> `{ "id" : "\"1234\"" }`
+* `_id` of type document e.g. { "hi" : "kafka", "nums" : [10.0, 100.0, 1000.0] } -> `{ "id" : "{\"hi\" : \"kafka\", \"nums\" : [10.0, 100.0, 1000.0]}" }`
```

Comment:
Would it be better to use a table with columns "Type", "MongoDB `_id` Value", "Key's payload"? If so, the last two lines below should be moved into the table as well.

Reply:
Fine for me if you prefer tabular layout. Here we go